pkg/executables/executables.go

```diff
@@ -88,9 +88,9 @@ func (e *executable) Close(ctx context.Context) error {
 func RedactCreds(cmd string, envMap map[string]string) string {
 	redactedEnvs := []string{}
 	for _, redactedEnvKey := range redactedEnvKeys {
-		if env, found := os.LookupEnv(redactedEnvKey); found {
+		if env, found := os.LookupEnv(redactedEnvKey); found && env != "" {
```

Comment:
I have var read as empty string, which would add mask before and after each char